tokens/project.json

```diff
@@ -40,46 +40,45 @@
 			},
 			"outputs": []
 		},
-		"format": {},
-		"lint": {},
-		"report": {
+		"diff": {
 			"cache": true,
 			"executor": "nx:run-commands",
 			"inputs": [{ "externalDependencies": ["@adobe/token-diff-generator", "@adobe/spectrum-tokens"] }],
 			"options": {
 				"commands": [
-					"tdiff report --old-token-version @adobe/spectrum-tokens@12.25.0 --new-token-version @adobe/spectrum-tokens@0.0.0-s2-foundations-20241121221506 --format markdown --output ./dist/tokens-diff.md"
+					"tdiff report --old-token-version @adobe/spectrum-tokens@{args.v1} --new-token-version @adobe/spectrum-tokens@{args.v2} --format markdown --output {projectRoot}/dist/tokens-diff.md"
 				],
 				"cwd": "{projectRoot}",
+				"forwardAllArgs": true,
 				"parallel": false
 			},
 			"outputs": [
 				"{projectRoot}/dist/tokens-diff.md"
 			]
 		},
+		"format": {},
+		"lint": {},
 		"style-dictionary": {
 			"dependsOn": ["clean"],
-			"executor": "@nxkit/style-dictionary:build",
-			"inputs": ["scripts"],
+			"executor": "nx:run-commands",
+			"inputs": [
+				"{projectRoot}/postcss.config.js",
+				"{projectRoot}/style-dictionary.config.js",
+				"{projectRoot}/utilities/style-dictionary.utils.js"
+			],
 			"options": {
-				"deleteOutputPath": false,
```

Comment:
The @nxkit tools don't seem to work in Windows _at all_ so moving to the style-dictionary CLI directly really helped improve how far our Windows workflow got. Still not enough to pass but much, much closer.